### ISSUE TITLE
Share button replacement position is incorrect on some computers

### ISSUE THREAD

**Issue by bates64** [GrannyCookies/scratchext2#5]
On some computers the css position-right is incorrect. I can reproduce this on my ChromeOS box.

![](http://i.imgur.com/beGZR8a.png)

**Comment by Felizolinha** [comment]
Could you provide steps on how to reproduce it?

**Comment by ghost** [comment]
You could make it recognize that it is a different system, so it should adjust itself accordingly.

**Comment by ghost** [comment]
At my Windows 7 I have the same problem

**Comment by bates64** [comment]
@Felizolinha I'm not sure how, I think it's that Lucida Grande isn't installed

**Comment by ghost** [comment]
Lucida Grande?

**Comment by bates64** [comment]
A font

**Comment by ghost** [comment]
Ah ok

**Comment by Felizolinha** [comment]
@GrannyCookies and that makes Scratch use another font which leads to different button sizes, is that correct?

**Comment by bates64** [comment]
No, the Flash in Scratch doesn't use a different font, but the website does.

**Comment by Felizolinha** [comment]
@GrannyCookies Do you have any idea on why the See project page button is bigger then?

**Comment by bates64** [comment]
?

**Comment by Felizolinha** [comment]
Do you have any clues on why it's positioning is wrong?

**Comment by bates64** [comment]
nope :/

**Comment by bates64** [comment]
try putting the two side by side?

**Comment by bates64** [comment]
![](http://i.imgur.com/gNKSTRc.png)
![](https://camo.githubusercontent.com/0c21adcae344b71a070054863db1843b2be2f567/687474703a2f2f692e696d6775722e636f6d2f6265475a5238612e706e67)

**Comment by bates64** [comment]
AHAH! I've figured it out! Yes, the font in the editor is different- it's wider!

**Comment by Felizolinha** [comment]
Okay, now I have to be sure what exactly is causing it to be wider :p

**Comment by GrannyCookies** [comment]
Get onto skype?
